src/interfaces/coral_web/src/components/Citations/Citation.tsx

```diff
@@ -146,9 +147,9 @@ export const Citation = React.forwardRef<HTMLDivElement, Props>(function Citatio
 
         <div className={cn('mb-4 flex items-center justify-between', { hidden: isSelected })}>
           <Text as="span" styleAs="caption" className="text-volcanic-800">
-            {uniqueDocuments.length} {pluralize('reference', uniqueDocuments.length)}
+            {uniqueDocuments.length} {pluralize('reference', uniqueDocuments.length)} from
```

Comment:
Would it be worth making this slightly more clear that it's XX references from XX urls/documents?

Reply:
I think we want to keep it generic because this could be a combination of docs, urls, tools, etc.